documentation/Using-the-SDK/Advanced/Disposable-Extensions.md

```diff
@@ -59,19 +64,20 @@ using (var engine = Engine.Create())
 } // all cookers, data sources, etc. cleaned up here
 ````
 
-Upon disposal, the Runtime will walk the graph of all loaded extensions and
-clean them up in order based on their dependencies. For example, if you have
-a `Composite Cooker` that dependes on a `Source Cooker`, then the `Composite
-Cooker` will be disposed _before_ the `Source Cooker`. Extensions are disposed
-before any of their dependencies are disposed.
-
-Due to the relationship between CustomDataSources and CustomDataProcessors, an 
-additional method has been added to the `ICustomDataSource` interface:
-DisposeProcessor. This method is invoked on the Custom Data Source right before
-a Custom Data Processor is to be cleaned up. This gives the Custom Data Source
-an opportunity to clean up any state it might have related to the processor.
-`CustomDataSourceBase` provides a default implementation of this method that
-does nothing.
+Upon disposal, the SDK runtime will determine the relationship between all
+loaded modules and clean them up in order based on their dependencies. For
+example, if you have a `CompositeCooker` that dependes on a `SourceCooker`,
+then the `CompositeCooker` will be disposed of _before_ the `Source Cooker`.
+In short, all extensions are disposed of before any of their dependencies are
+disposed.
+
+Due to the relationship between a `CustomDataSource` and a
+`CustomDataProcessor`, an additional method has been added to the
```

Comment:
"has been added" makes sense in PRs but not docs. Maybe change "an additional... " to "`ICustomDataSource` has a `DisposeProcessor` method."